sematic/db/models/run.py

```diff
@@ -145,6 +146,13 @@ class Run(HasUserMixin, HasOrganizationMixin, Base, JSONEncodableMixin):
 
     # Relationships
     root_run: "Run" = relationship("Run", remote_side=[id], lazy="select")
+    pipeline_run: Resolution = relationship(
+        "Resolution",
+        foreign_keys=[root_id],
```

Comment:
Good question, I guess based on the current usage, there is no need to add that. This attribute is simply added there to "document" a relationship, so we can automatically construct a "WHERE" condition. This significance is represented by the "viewonly=True" option. The synchronization offered by `back_populates()` is not of any use for us at the moment. And I feel it might be somewhat dangerous to add that, in case accidental synchronization would be caused.